system/lib/sbrk.c

```diff
@@ -35,6 +35,8 @@
 }
 
 void *sbrk(intptr_t increment) {
+  // Enforce preserving a minimal 4-byte alignment for sbrk.
+  increment = (increment + 3) & ~3;
```

Comment:
This breaks the use case of `sbrk(0)` being used to find the current break point, I think?

Reply:
Oh no wait, it's still zero then, nm...